client/src/app/services/chart.service.ts

```diff
@@ -525,11 +539,37 @@ export class ChartService {
   }
   //#endregion
 
+  //#region WINDOW AND TIMEFRAME OPERATIONS
```

Comment:
consider, we remove timeframe picker, but still need to resize array accordingly

Reply:
This should occur dynamically with dejittering delay with window resizing in the browser only

Reply:
I don’t see where you added a 300ms debounce time